packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PostConfirmation/cloudformation-templates/PostConfirmation.json.ejs

```diff
@@ -38,16 +38,16 @@
         "env": {
             "Type": "String"
         }<%if (props.dependsOn && props.dependsOn.length > 0) { %>,<% } %>
-    <% if (props.dependsOn) { %>
-    <% for(var i=0; i < props.dependsOn.length; i++) { %>
-    <% for(var j=0; j < props.dependsOn[i].attributes.length; j++) { %>
+    <% if (props.dependsOn) -{ %>
```

Comment:
There are a few occurrences of `-{ %>`. Are these supposed to be `{ -%>`?

Reply:
Yes it should be, thanks for catching that!